src/Compilers/CSharp/Portable/Parser/LanguageParser.cs

```diff
@@ -11173,7 +11182,7 @@ private ParameterSyntax ParseLambdaParameter(bool isFirst, ref bool hasTypes)
                     refOrOutOrParams = this.EatToken();
                 }
 
-                paramType = this.ParseType(true);
+                paramType = this.ParseType();
```

Comment:
Actually, I don't think we need that mode any longer. I'll investigate and remove it if true.

---
In reply to: [79675184](https://github.com/dotnet/roslyn/pull/13814#discussion_r79675184) [](ancestors = 79675184)